src/breadcrumbs/breadcrumbs.scss

```diff
@@ -18,47 +18,67 @@
   user-select: none;
 }
 
-@mixin iui-breadcrumbs-item {
+@mixin iui-breadcrumbs-item($breadcrumbs-height: $iui-component-height) {
+  --_iui-breadcrumbs-item-separator-svg: url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16'><path d='m4.7 0-1.4 1.4 6.6 6.6-6.6 6.6 1.4 1.4 8-8z' /></svg>");
+
   display: flex;
-  line-height: $iui-component-height;
-  height: $iui-component-height;
-  max-width: $iui-3xl * 2;
-  margin: 0 $iui-sm;
+  align-items: center;
+  line-height: $breadcrumbs-height;
+  height: $breadcrumbs-height;
 
-  a {
-    @include iui-anchor;
+  &:not(:last-child)::after {
+    flex-shrink: 0;
+    width: $iui-icons-small;
+    height: $iui-icons-small;
+    margin: 0 $iui-xxs;
+
+    // #region IE support & future support
+    content: url("data:image/svg+xml,%3Csvg viewBox='0 0 16 16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m4.7 0-1.4 1.4 6.6 6.6-6.6 6.6 1.4 1.4 8-8z' fill='%23000' fill-opacity='0.4' /%3E%3C/svg%3E");
+    line-height: 14px;
+
+    @supports (mask: var(--_iui-breadcrumbs-item-separator-svg)) {
+      content: '';
+      background-color: var(--iui-icons-color);
+      mask: var(--_iui-breadcrumbs-item-separator-svg);
+    }
+    // #endregion IE support & future support
+
+    @media (forced-colors: active) {
+      background-color: CanvasText;
+    }
   }
+}
 
-  .iui-button {
-    @include iui-button-borderless;
-    margin: 0 (-$iui-s - 1);
+@mixin iui-breadcrumbs-text {
+  margin: 0 ($iui-s + 1);
```

Comment:
9px? Something is wrong

Reply:
Our buttons have `$iui-s` padding & 1px border.
![Screen Shot 2022-04-04 at 2 44 14 PM](https://user-images.githubusercontent.com/849817/161610706-e4d831ad-f061-4199-b5be-1c44690d57b7.png)

I added this here so the hyperlink & button text align.
![Screen Shot 2022-04-04 at 2 46 55 PM](https://user-images.githubusercontent.com/849817/161611163-e8fff9e6-f690-415f-a1bc-b714f6eed466.png)

Reply:
Went the opposite approach and removed the border, so now only `$iui-s` is used.